nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -728,6 +731,21 @@ def allreduce_sequence_parallel_gradients(self):
         for buf, synced in zip(grads, torch._utils._unflatten_dense_tensors(coalesced, grads)):
             buf.copy_(synced)
 
+    def allreduce_fsdp_sharding_omitted_gradients(self):
```

Comment:
Doesn't have to be this PR, but shouldn't these calls go into an fsdp strategy rather than the model code? Maybe @athitten , you can address this is in a future PR.